src/Microsoft.Diagnostics.Monitoring.EventPipe/Triggers/EventCounter/EventCounterTrigger.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tracing;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe.Triggers.EventCounter
```

Comment:
I vaguely recall some discussions around this, but would it be reasonable to keep some/all concrete trigger implementations in the dotnet-monitor repo and the infrastructure in the diagnostics repo?

Reply:
There's some desire to have the other dotnet tools to use these triggers (e.g. https://github.com/dotnet/diagnostics/issues/2478), so I think most if not all of the triggers would end up here. If there are triggers that depend on something that is specific to dotnet-monitor (e.g. uses some service, additional settings, etc), then that should go to dotnet-monitor.